src/libs/actions/IOU.ts

```diff
@@ -406,7 +406,7 @@ function setMoneyRequestCreated(transactionID: string, created: string, isDraft:
 
 function setMoneyRequestCurrency(transactionID: string, currency: string, isEditing = false) {
     const fieldToUpdate = isEditing ? 'modifiedCurrency' : 'currency';
-    Onyx.merge(`${ONYXKEYS.COLLECTION.TRANSACTION_DRAFT}${transactionID}`, {[fieldToUpdate]: currency});
+    Onyx.merge(`${ONYXKEYS.COLLECTION.TRANSACTION_DRAFT}${transactionID}`, {[fieldToUpdate]: currency, currencyUpdatedTime: new Date().getTime()});
```

Comment:
Using a date here is hacky. And another concern is that once this value is set it won't be unset making it useless and the conditions that check it will always be true which could lead to bugs

Reply:
@s77rt  This will not be a problem because it's updated with `currency`.

Reply:
if we want to reset this value we can reset it in `initMoneyRequest` function.